### PR TITLE
remove kustomize refs, overhaul provider guide

### PR DESCRIPTION
**Pull request by rsum1**
fixes #1204

### CODE REVIEW

File: guides/provider/README.md [hunk 1]
```diff
@@ -1,330 +1,458 @@
...
+You need a working Ansible installation as well, with the additional Python dependencies specified in the `requirements.txt` document. It is recommended you install this using a Python virtual environment created with the `virtualenv` tool.
 
+```
+virtualenv --python=python3 venv
+```
+```
+. venv/bin/activate
+```
+```
+sudo pip3 install -r requirements.txt
```
Comment (hydrogen18) [inline guides/provider/README.md:50]: After doing `activate` you should not need to use `sudo` in any context for running python. The `pip` command installs things into the virtual environment, not the machines root filesystem

File: guides/provider/README.md [hunk 2]
```diff
@@ -1,330 +1,458 @@
...
+2. Increase `kube_pods_subnet` to allow more pods in the kubernetes cluster
+3. Increase `kube_service_addresses` since more services are expected to be running.
 
-```sh
-akash tx provider create provider.yaml --from $AKASH_PROVIDER_KEY --home=$AKASH_HOME --keyring-backend=$AKASH_KEYRING_BACKEND --node=$AKASH_NODE --chain-id=$AKASH_CHAIN_ID
-```
+In this example, these values can be configured in `kubespray/inventory/defi/group_vars/k8s_cluster/k8s-cluster.yml`
 
-### Configure provider services client
+<!-- TODO - add notes about adding CRD -->
```
Comment (hydrogen18) [inline guides/provider/README.md:178]: This is explained below when `kubectl` is used to invoke `crd.yaml` , so we can drop the TODO

File: guides/provider/README.md [hunk 3]
```diff
@@ -1,330 +1,458 @@
...
+There are 3 important public hostnames that a provider must configure.
 
-```sh
-curl -s -o deployment.yaml \
-  https://raw.githubusercontent.com/ovrclk/akash/master/_docs/examples/provider/deployment.yaml
-```
+The **provider host** is the publicly accessible hostname of the provider. This is specified in the configuration file using the `host` key when `akash tx provider create` or `akash tx provider update` is ran. This value is stored on the blockchain. It is used whenever a lease owner needs to communicate directly when the provider for things such as sending a manifest or getting a lease status.
 
-### Customize SDL
+The **cluster ingress host** is the publicly accessible hostname of the Kubernetes cluster. The Kubernetes cluster hosts an Ingress Controller, which is one way that leases in the cluster may be exposed to the outside world. At this time only HTTP is supported for the Ingress Controller. The hostname should resolve to an IP which directs traffic to the Kubernetes ingress controller IP on your network. This value is specified using the `--deployment-ingress-domain` switch. It is not stored on the blockchain.
```
Comment (hydrogen18) [inline guides/provider/README.md:267]: Reference the node with applied label `akashRole=ingress` here

File: guides/provider/README.md [hunk 4]
```diff
@@ -1,330 +1,458 @@
...
+### Starting the Provider
+
+#### Mandatory provider configuration
+
+You must configure the following parameters on the command line when starting the provider
+
+* `--home`
+* `--chain-id`
+* `--keyring-backend` - always set to `test`
+* `--from`
```
Comment (hydrogen18) [inline guides/provider/README.md:330]: This guy is listed twice

File: guides/provider/README.md [hunk 5]
```diff
@@ -1,330 +1,458 @@
...
+### Create Kubernetes Ingress Controller 
 
-```sh
-kubectl kustomize .
-```
+Akash requires that a Kubernetes ingress controller be created. You must load this file by downloading it and applying it 
 
-### Install kubernetes configuration
+Files:
+1. [https://raw.githubusercontent.com/ovrclk/akash/master/_run/ingress-nginx.yaml]
```
[the remark this answers is not in the document]
Comment (hydrogen18) [reply guides/provider/README.md:224]: This is caused by this PR https://github.com/ovrclk/akash/pull/1257

**Review by hydrogen18 — APPROVED** [review]

**Review by boz — APPROVED** [review]
:+1:

**Pull request merged**